src/entities/friendships.rs

```diff
@@ -169,7 +184,7 @@ impl FriendshipRepositoryImplementation for FriendshipsRepository {
         let mut query =
             "SELECT * FROM friendships WHERE (address_1 = $1) OR (address_2 = $1)".to_owned();
 
-        if include_inactive {
+        if only_active {
```

Comment:
Are we querying for inactive friends somewhere in the codebase or planning for it?

Reply:
We were going to use it in the room events endpoint but in the end the logic was simplified to prevent the need, and I don't think that in the near future we'll need it